theurgy.lic

```diff
@@ -18,9 +15,12 @@ class TheurgyActions
   def initialize(settings)
     @settings = settings
     @hometown = @settings.hometown
-    @data = get_data('theurgy')[@hometown]
+    all_theurgy_data = get_data('theurgy')
+    @data = all_theurgy_data[@hometown]
+    @immortal_to_aspect = all_theurgy_data['immortal_to_aspect']
     @safe_room = @settings.safe_room
     @immortal_aspect = @settings.immortal_aspect
+    @last_favor_aspect = last_favor_aspect
```

Comment:
Why check this here at `initialize` time, when they may not be doing the `clean_anloral` ritual? I.e. what do you think about moving this down into the ritual function?

Reply:
Done.